### ISSUE TITLE
Custom parameters

### ISSUE THREAD

**Issue by bbbales2** [StochSS/stochss#530]
Setting parameter k2 equal to sqrt(0.03) causes a 'None' to be injected into the StochKit2 xml file and the simulation fails.

This is an error

**Comment by bbbales2** [comment]
This is causing the error in https://github.com/StochSS/stochss/issues/529. https://github.com/StochSS/stochss/issues/529 is still a separate issue because the failed status is reported incorrectly

**Comment by bbbales2** [comment]
Now I don't think sqrt(0.03) should work. I fixed the problem that was turning sqrt(0.03) into None (https://github.com/StochSS/stochss/commit/8a11f9f93c762d7c6a02c5cbce1ce7ac08794d61) and the model still didn't run.

I found an e-mail exchange I had with Sheng/Brian 9/24/2015. Summary here:

">> (Brian) In mass-action reaction, can parameters be functions of other parameters?

Well they can be like four-function computations of other parameters. */+-. C functions and Python functions do not work.

>> I think that having functions in parameters is not something we should support, and it should throw an error.

For parameter sweep what I do is just set the minimums and maximums to zero when a NaN is used and tell them I did that -- it's the user's responsibility to fix it."

I still think changing the parameter to a None was a mistake, so I committed the code, but otherwise I don't think this is a bug and should be closed.
